src/core/ready.js

```diff
@@ -1,6 +1,7 @@
 define( [
 	"../core",
 	"../var/document",
+	"../core/readyException",
```

Comment:
Oh, i see https://github.com/jquery/api.jquery.com/issues/942, we need a pr for it though? Before we can release?